src/bundle/Resources/config/serializer.yaml

```diff
@@ -29,10 +29,6 @@ services:
         tags:
             - { name: ibexa.rest.serializer.normalizer, priority: -400 }
 
-    Ibexa\Rest\Output\Normalizer\JsonSerializableNormalizer:

```

Comment:
Don't forget to register the Symfony class as a service.

Reply:
Fine by me: https://github.com/ibexa/rest/pull/160/commits/f10b4daa0accabb4673ac47374285a8131c2a0a6.